runtime/sema/errors.go

```diff
@@ -2303,6 +2303,22 @@ func (e *InvalidResourceTransactionParameterError) Error() string {
 
 func (*InvalidResourceTransactionParameterError) isSemanticError() {}
 
+// InvalidNonStorableTransactionParameterTypeError
+
+type InvalidNonStorableTransactionParameterTypeError struct {
+	Type Type
+	ast.Range
+}
+
+func (e *InvalidNonStorableTransactionParameterTypeError) Error() string {
+	return fmt.Sprintf(
+		"transaction parameter must not be storable: `%s`",
```

Comment:
`must be storable` or `is not storable` ?

Reply:
Good catch, this was copy and pasted from above (`InvalidResourceTransactionParameterError`) and I rephrased incorrectly 👍 

```suggestion
		"transaction parameter must be storable: `%s`",
```